src/components/PhotoUpload/PhotoUploadInstructionsToolbar.js

```diff
@@ -0,0 +1,312 @@
+        }
+      }
+    }
+  }
+
+  > footer {
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+  }
+
+  input.hidden-file-input {
+    visibility: hidden;
+    display: none;
+  }
+`;
+
+export default class PhotoUploadInstructionsToolbar extends React.Component<Props, State> {
+  props: Props;
+
+  state = {
+  };
+
+  inputField: ?HTMLInputElement;
+  backLink: ?React.ElementRef<typeof CloseLink>;
+  goButton: ?React.ElementRef<'button'>;
+
+  componentDidMount() {
+  }
+
+  componentDidUpdate(prevProps: Props, prevState: State) {
+  }
+
+  renderCloseLink() {
+    return <button
+      className='close-link'
+      // translator: Button caption in photo upload Instructions dialog
+      aria-label={t`Back`}
+      onClick={this.onClose}
+      ref={backLink => this.backLink = backLink}
+    >← </button>;
+  }
+
+
+  onFileInputChanged = (event: SyntheticEvent<HTMLInputElement>) => {
+    const input = event.currentTarget;
+    var files = input.files;
+
+    if (!files || files.length === 0) {
+      if (this.props.onClose) {
+        this.props.onClose();
+      }
+    } else {
+      if (this.props.onCompleted) {
+        this.props.onCompleted(files);
+      }
+    }
+  }
+
+  onClose = (event: UIEvent) => {
+    if (this.props.onClose) {
+      this.props.onClose();
+      event.preventDefault();
+    }
+  }
+
+  render() {
+    return (
+      <StyledToolbar
+        className='photoupload-instructions-toolbar'
+        hidden={this.props.hidden}
+        isSwipeable={false}
+        isModal
+      >
+        <header>
+          {this.renderCloseLink()}
+          <h3>{t`The following images...`}</h3>
+        </header>
+        <section>
+          <ul>
+            <li className='with-checkmark'>
+              <span><StyledCheckmarkIcon color={colors.linkColor} /><p>{t`...are interesting for accessibility.`}</p></span>
+              <ul className='photo-examples'>
+                <li>
+                  <div className='placeholder-image entrance-image'></div>
+                  <small>{t`entrances`}</small>
+                </li>
+                <li>
+                  <div className='placeholder-image sitemap-image'></div>
+                  <small>{t`access plans`}</small>
+                </li>
+                <li>
+                  <div className='placeholder-image toilet-image'></div>
+                  <small>{t`toilets`}</small>
+                </li>
+              </ul>
+            </li>
+            <li className='with-checkmark'>
+              <span><StyledCheckmarkIcon color={colors.linkColor} /><p>{t`...I made them myself. `}</p></span>
+              <small>{t`Therefore I can give Wheelmap.org unlimited rights of use in form of a Creative Commons-License (CC).`}</small>{/* TODO: add link to CC */}
+            </li>
+            <li className='with-checkmark'>
+              <span><StyledCheckmarkIcon color={colors.linkColor} /><p>{t`...do not contain any persons.`}</p></span>
```

Comment:
Would use 'people' here because the bullet point can still be informal and friendly, and use `persons` in the law-style text below.

Reply:
@nero-still can you look at this?